src/backend/EdFi.DataManagementService.Backend.Postgresql/Deploy/Scripts/0001_Create_Documents_Table.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE IF NOT EXISTS documents (
+  id BIGINT GENERATED ALWAYS AS IDENTITY(START WITH 1 INCREMENT BY 1),
+  document_partition_key SMALLINT NOT NULL,
+  document_uuid CHAR(36) NOT NULL,
+  resource_name VARCHAR(256) NOT NULL,
+  edfi_doc BYTEA NOT NULL,
+  PRIMARY KEY (document_partition_key, id)
+) PARTITION BY HASH(document_partition_key);
+
+CREATE TABLE documents_01 PARTITION OF documents FOR VALUES WITH (MODULUS 15, REMAINDER 0);
+CREATE TABLE documents_02 PARTITION OF documents FOR VALUES WITH (MODULUS 15, REMAINDER 1);
+CREATE TABLE documents_03 PARTITION OF documents FOR VALUES WITH (MODULUS 15, REMAINDER 2);
+CREATE TABLE documents_04 PARTITION OF documents FOR VALUES WITH (MODULUS 15, REMAINDER 3);
+CREATE TABLE documents_05 PARTITION OF documents FOR VALUES WITH (MODULUS 15, REMAINDER 4);
+CREATE TABLE documents_06 PARTITION OF documents FOR VALUES WITH (MODULUS 15, REMAINDER 5);
+CREATE TABLE documents_07 PARTITION OF documents FOR VALUES WITH (MODULUS 15, REMAINDER 6);
+CREATE TABLE documents_08 PARTITION OF documents FOR VALUES WITH (MODULUS 15, REMAINDER 7);
+CREATE TABLE documents_09 PARTITION OF documents FOR VALUES WITH (MODULUS 15, REMAINDER 8);
+CREATE TABLE documents_10 PARTITION OF documents FOR VALUES WITH (MODULUS 15, REMAINDER 9);
+CREATE TABLE documents_11 PARTITION OF documents FOR VALUES WITH (MODULUS 15, REMAINDER 10);
+CREATE TABLE documents_12 PARTITION OF documents FOR VALUES WITH (MODULUS 15, REMAINDER 11);
+CREATE TABLE documents_13 PARTITION OF documents FOR VALUES WITH (MODULUS 15, REMAINDER 12);
+CREATE TABLE documents_14 PARTITION OF documents FOR VALUES WITH (MODULUS 15, REMAINDER 13);
+CREATE TABLE documents_15 PARTITION OF documents FOR VALUES WITH (MODULUS 15, REMAINDER 14);
```

Comment:
It looks weird on the SQL Server side but there are actually 16 partitions.

Reply:
@simpat-adam 
Why are we not using partition scheme and partition function to create partitions ? https://github.com/Ed-Fi-Alliance-OSS/Project-Tanager/blob/main/POC-Applications/POC.Databases/Artifacts/MsSql/00002-CreateDMSTables.sql

Reply:
This is the postgres world

Reply:
Oops. I added the partition and started the numbering with 00.